src/components/DraftSection.vue

```diff
@@ -67,11 +102,52 @@ const changeSelectedHero = (hero: IHero) => {
   state.selectedHero = hero;
 };
 
-const resetSelectedHero = () => {
-  state.selectedHero = null;
+const firstPickChosen = (isRadiantFirstPick: boolean) => {
+  state.showSelectFirstPickModal = false;
+  if (isRadiantFirstPick) {
+    state.firstPickPlayer = playerStore.radiantPlayer;
+    state.secondPickPlayer = playerStore.direPlayer;
+  } else {
+    state.firstPickPlayer = playerStore.direPlayer;
+    state.secondPickPlayer = playerStore.radiantPlayer;
+  }
 };
 
-const startGame = () => {};
+const heroChosen = (hero: IHero | null) => {
```

Comment:
hideous logic, we need to make early return (like if `!isPick.value` then return)
besides that, lets separate it to minor functions for readability
probably will need to communicate on that together